### ISSUE TITLE
Elementor Pro License activation for Multisite for all sites

### ISSUE THREAD

**Issue by navdih** [elementor/elementor#10634]
## Prerequisites

- [x] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [x] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**
I wuld like to include Elementor Pro in Multisite. The problem is that the plugin's license activation can't be done in Network Admin. At the moment the only option is to log into each of the sites and activate the license by hand one by one. This is No Go option when clients paid for the site online (on Multisite network) and don't have this option activated right at start and need to wait until Super Admin activates it. 

**Describe the solution you'd like**
Elementor Pro Licence activation is done only once in Network Admin for all sites. Because there is no Elementor menu in Netork Admin, you should add it including License submenu /Elementor/License/. You culd also put this option under /Settings/Elementor/ submenu.

**Describe alternatives you've considered**
/

**Additional context**
Beaver Builder is having this opton for Multisite activation https://wpbeaveraddons.com/docs/general-information/getting-started/activate-license-key-on-wordpress-multisite/
FOr us ELementor is no1 WP builder and it should also have this option included in next update. Thank you.

**Comment by arielk** [comment]
I merged this issue with another issue https://github.com/elementor/elementor/issues/8876 in the same subject so we could deal with it more efficiently. So please let's continue the conversation there.

Thanks for your feedback!

**Comment by webenadam** [comment]
+1

**Comment by gene158** [comment]
+1
